src/lib/utils.js

```diff
@@ -26,16 +34,8 @@ export const isMobile = () => /Mobile|iP(hone|od|ad)|Android|BlackBerry|IEMobile
  * @param {string} event.title
  * @returns {string} Google Calendar Event URL
  */
-const googleShareUrl = ({
-  description,
-  endDatetime,
-  location,
-  startDatetime,
-  title,
-}) =>
-  `https://calendar.google.com/calendar/render?action=TEMPLATE&dates=${
-    startDatetime
-  }/${endDatetime}&location=${location}&text=${title}&details=${description}`;
+const googleShareUrl = ({ description, endDatetime, location, startDatetime, title }) =>
```

Comment:
Let's not change the original formatting. Adds lots of extra changes to the PR and makes this file not consistent with the rest of them.